### PR TITLE
[ESSI-412] Toggle Show / Hide for Child Items button.

### PR DESCRIPTION
**Pull request by andjsmit**

### CODE REVIEW

File: app/assets/javascripts/work_items.es6 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export default class WorkItems {
+  constructor() {
+    this.toggle_show_items()
+  }
+
+  toggle_show_items(){
+    let button = $("#work-items-button")
+    let show = I18n.t('hyrax.works.form.show_child_items')
```
Comment (randalldfloyd) [inline app/assets/javascripts/work_items.es6]: Should there also be a translation by default in `en.hyrax` for this?  When I desk check I get `missing "en.hyrax.works.form.hide_child_items" translation`

Comment (andjsmit) [reply app/assets/javascripts/work_items.es6]: Ooops. Missed committing locale file.

Comment (randalldfloyd) [reply app/assets/javascripts/work_items.es6]: Works locally now.

**Comment by randalldfloyd** [comment]
Merging with CI failure.  Changes work locally and the failure is the pesky LDP conflict.

**Pull request merged**